src/classes/RD2_RecurringDonations_TDTM.cls

```diff
@@ -382,14 +382,42 @@ public class RD2_RecurringDonations_TDTM extends TDTM_Runnable {
         RD2_QueueableService.EvaluateInstallmentOpportunities service =
             new RD2_QueueableService.EvaluateInstallmentOpportunities(rdIds, isScheduleChangedById);
 
-        if (Limits.getQueueableJobs() < Limits.getLimitQueueableJobs()) {
-            System.enqueueJob(service);
+        if (triggerAction == TDTM_Runnable.Action.AfterInsert && evaluateOppsSynchronously()) {
```

Comment:
Previously the queueable limit was being checked on both insert and update, but currently it is being checked for the insert only.